spec/system/homepage_spec.rb

```diff
@@ -10,11 +10,23 @@
     expect(page).not_to have_css(".homepage-inverse-header__title")
   end
 
-  it "renders the search with autocomplete component with the correct source URL" do
-    visit "/"
+  context "search autocomplete" do
```

Comment:
This should be failing our RSpec linting (turns out this project doesn't have our rspec rules installed 😢 https://github.com/alphagov/frontend/blob/63d532445eb2ebf15993f7de9850184acd231ea6/.rubocop.yml#L4) by not having a "when" or "with" at the start - https://rspec.rubystyle.guide/#context-descriptions

I'd imagine to make it more idiomatic RSpec you might want to refactor this two contexts:

context "when search with autocomplete is enabled"
context "when search with autocomplete is disabled"